app/views/health/careplans/_pctp_signatures.haml

```diff
@@ -1,31 +1,62 @@
 %article.health-care-plan__responsible-parties.l-print__section--page
   %h1 Responsible Parties Signatures
 
-  %dl.l-print__item
-    %dt.inline Patient:
-    %dd.inline= @patient.name
-    %dd.l-print__signature &nbsp;
-    %dt.inline Signed On:
-    %dd.inline= @careplan.patient_signed_on&.to_date
+  .signature__section
+    .signature__section__row
+      .signature__label Patient:
+      .signature__value= @patient.name
+    .signature__section__row
+      .signature__section__signature
+        .signature__label Patient Signature:
+        .signature__line
+      .signature__section__signedOn
+        .signature__label Signed On:
+        - if @careplan.patient_signed_on&.to_date
+          .signature__value= @careplan.patient_signed_on&.to_date
+        - else
+          .signature__line
 
-  %dl.l-print__item
-    %dt.inline Representative:
-    %dd.inline= @careplan.representative&.full_name
-    %dd.l-print__signature &nbsp;
-    %dt.inline Signed On:
-    %dd.inline= @careplan.representative_signed_on
+  .signature__section
+    .signature__section__row
+      .signature__label Representative:
+      - if @careplan.representative&.full_name
+        .signature__value= @careplan.representative&.full_name
+      - else
+        .signature__line.signature__line-name
+    .signature__section__row
+      .signature__section__signature
+        .signature__label Representative Signature:
+        .signature__line
+      .signature__section__signedOn
+        .signature__label Signed On:
+        - if @careplan.representative_signed_on
+          .signature__value= @careplan.representative_signed_on
```

Comment:
I don't know if `representative_signed_on` ever gets set.

Reply:
sounds like this is not used so maybe we should remove it

Reply:
I could swear we implemented this, but don't see it in the code, let's remove it.

Reply:
Ok. Should we still print the representatives name if `@careplan.representative` is present? ie if a representative was selected in the top right input element here:
<img width="975" alt="Screen Shot 2022-09-27 at 12 55 26 PM" src="https://user-images.githubusercontent.com/5333982/192821035-c501b447-37a3-4b42-9f39-7c2eb3b6f99a.png">

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2487#discussion_r982578746 this screenshot shows what it would look like

Reply:
OK, representative should only show if there's a name (in the PDF version, and it should appear directly below the "Patient" line.  So it would be:
```
Patient: Arthur Weasly
Representative: He who must not be named

Patient Signature: ....
```

We don't need a separate signature line for the representative.

Reply:
Ok thanks! Updated:
![Screen Shot 2022-09-28 at 12 52 14 PM](https://user-images.githubusercontent.com/5333982/192842753-290c7fd0-a21d-4a11-b950-0463d4a4d680.png)

Reply:
Looks great!